README.md

```diff
@@ -9,6 +9,11 @@ See the Spectator [documentation] for an overview of core concepts and details o
 
 Supports Python >= 3.5, which is the oldest system Python 3 available on our commonly used OSes.
 
+Note that there is a risk of deadlock if you are running Python 3.6 or lower and using 
```

Comment:
I would add a cross-doc link to the Concurrent Usage stuff from here (i.e. see ... for more details).